### ISSUE TITLE
Remove the checkmark for honest marketing for three providers

### ISSUE THREAD

**Issue by MooseExplorer637** [techlore/VPN-reviews#62]
Ivacy, WeVPN, and PIA do not deserve the check for honest marketing. Here are the reasons.

Ivacy: Their reasoning for using a VPN is just straight-up fearmongering. Take a look at [this page](https://www.ivacy.com/what-is-vpn/). I could give some examples here, but the page does more justice to their disgusting marketing. In my opinion, Ivacy is probably the worst offender when it comes to services that have received the checkmark.

WeVPN: Their "Do I really need a VPN" answer tells quite a lot about them as a company: "Every hour, more than 1,000 people have their identities stolen in the US alone, often on the internet. Using a reliable VPN service like WeVPN is one of your best defenses against online identity theft and fraud. Additionally, WeVPN blocks ads, trackers and malware from infecting your device and its powerful VPN encryption and privacy features prevent websites, hackers, government agencies and your ISP from tracking your every move, making it a must have tool for anyone that uses the internet." So, according to them, a VPN is a must-have tool for protecting against identity thefts. Again, they should be definitely removed from the list.

PIA: "Keep your data encrypted for extra security. Don’t let anyone get their hands on unencrypted emails, pictures, banking details, or anything else best kept private." It probably goes without saying that VPN is not really useful for this kind of purpose that HTTPS is already doing.

**Comment by henryistaken** [comment]
Agreed with Ivacy & WeVPN - will remove them in an upcoming update 👍

Regarding PIA, this feels a bit nitpicky as *technically* it's accurate. If someone is browsing via HTTP, then this is still a legitimate (though yes, highly unlikely) usecase. Funny enough I typed out some reasoning as to why I felt PIA did an okay job with marketing when I stumbled on this from their homepage:

> Your online activities can’t be tracked when you’re using a top-shelf VPN. PIA’s VPN servers reroute and encrypt your internet traffic and mask your IP address in real-time. By using PIA you can make sure ISPs, government bodies, and cybercriminals can’t monitor your online activities and pinpoint your location.

> Concealing your IP address also prevents websites and trackers from identifying your online activities. This way, you’ll stay incognito, and bypass network blocks while traveling. A VPN is crucial to improving your online anonymity and freedom.

Now **this** is where they pushed a boundary..._"Your online activities can’t be tracked when you’re using a top-shelf VPN"_  So on those grounds I'll remove PIA as well in the same update. Will update this issue once it's completed.

**Comment by henryistaken** [comment]
https://github.com/techlore/VPN-reviews/commit/826c31d5c55fe7cc0e7f9dbcea4ed9d22c637da5
Fixed in the latest update
